autopush/router/apnsrouter.py

```diff
@@ -157,7 +158,7 @@ def _route(self, notification, router_data):
                                        self._base_tags,
                                        application=rel_channel,
                                        reason="connection_error"))
-        except HTTP20Error:
+        except (HTTP20Error, socket.error):
```

Comment:
This makes the SSLError block below dead code (SSLError derives from socket.error). Go ahead and kill it